### ISSUE TITLE
Enable Speech - Render error in Emulator

### ISSUE THREAD

**Issue by richross** [microsoft/botframework-solutions#3338]
/clients-and-channels/tutorials/enable-speech/4-integrate-speech-channel/

I'm trying to enable speech in Virtual Agent.  I did the following steps.
**Create the VA**
Ran through the install of the VA tools and projects
Created a new bot VA project in VS 2019
Deployed the Azure components via the PS script
Got the bot working in the emulator. 
 
**Tutorial - Enable Speech**
Followed the Enable Speech path under the Next Steps
Created the Speech Service
Confirmed all of the configuration settings for the app and services including the streaming endpoint and websockets.
Ran the bot in the Emulator.  As soon as the conversation is started, the adaptive card is displayed, I am asked for my name, and then there is a 'non-bot' message, possibly from the emulator stating "Render error. Please check the console or contact the bot developer." all in red text.
I included an image of the emulator.  The emulator is mostly unresponsive at this point and needs to be shut down.  I can enter text but cannot submit the messages.

Thanks!
![image](https://user-images.githubusercontent.com/773157/81299354-16a04700-9044-11ea-8061-60d5d88b3e71.png)

**Comment by ToddVan** [comment]
I can confirm this issue. Same steps, same behavior.

**Comment by ryanlengel** [comment]
Hi @richross, I ran through the same steps and am unable to reproduce on my end (Emulator `v4.8.1-126122`). Is this against a bot published on Azure or running locally? Do the console/App Insights logs have anything useful?

![image](https://user-images.githubusercontent.com/43043272/81349196-60088a80-9074-11ea-8825-c874302b0202.png)

**Comment by richross** [comment]
Hi Ryan - yes running 4.8.1 here as well for the emulator.  
I can get the virtual agent to run.  Were you able to get the Enable Speech tutorial running?  That is where I get the issue.  This happens with a bot running locally and in Azure.

**Comment by ryanlengel** [comment]
Yes I ran the Enable Speech tutorial successfully. Can you test against the [Windows Voice Assistant Client](https://github.com/Azure-Samples/Cognitive-Services-Voice-Assistant/blob/master/clients/csharp-wpf/README.md) and share the results? It will help to validate if the same issue occurs on another Direct Line Speech client and if it related to the bot. If there is an issue there it may provide better output to debug.

**Comment by richross** [comment]
Yes, this works in the client app.

**Comment by ryanlengel** [comment]
Got it, thanks for checking! In that case this seems like a bug with BF Emulator, I am transferring to their repository.
